tools/apache_rat_license_check/rat-excludes

```diff
@@ -0,0 +1,109 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# Version control
+.github
+.gitignore
+.gitattributes
+.gitmodules
+
+# The Apache MXNET (incubating) project contains subcomponents with separate
+# copyright notices and license terms. Your use of the source code for the
+# these subcomponents is subject to the terms and conditions of the following
+# licenses. If not stated otherwise, their copyright notices and license terms
+# are available at the path of the subcomponent.
+3rdparty
+clipboard.js
+
+# Licenses
+licenses/*
+LICENSE.binary.dependencies
+
+# Generated files during build
+.buildinfo
+Gemfile.lock
+_build/*
+_static/*
+_site/*
+_api/*
+_includes/*
+build/*
+latex/*
+target/*
+site/*
+xml/*
+DartConfiguration.tcl
+.*\.egg-info
+.*\.t
+
+# Files generated by Cython
+core.cpp
+symbol.cpp
+ndarray.cpp
+
+# Sphinx themes
+themes/*
+
+# Binary or data files
+.*\.ipynb
+.*\.pyc
+.*\.so
+.*\.json
+.*\.txt
+.*\.svg
+.*\.lst
+.*\.lds
+.*\.in
+.*\.diff
+.*\.edl
+.*\.md5
+.*\.csv
+.*\.log
+.*\.interp
+.*\.tokens
+.*\.cPickle
```

Comment:
Do we need to keep these files in source control?

Reply:
I think we still have some of such data files in the code base and I'd like to limit the scope of the change.